packages/vertica-nodejs/mochatest/integration/client/vertica-connection-params-tests.js

```diff
@@ -62,10 +62,10 @@ describe('vertica protocol_version connection parameter', function () {
   })
 
   it('provides a maximum value for the protocol version used by the server', function(done) {
-    const client = new vertica.Client({client_label: 'pvTest'}) // make easy to find session
+    const client = new vertica.Client()
     client.connect()
-    client.query("SELECT effective_protocol from sessions where client_label = 'pvTest'", (err, res) => {
-      if (err) assert(false)
+    client.query("SELECT effective_protocol FROM sessions WHERE session_id=(SELECT current_session())", (err, res) => {
```

Comment:
Same thing here. This is cleaner than the check for client label, but no need for a subquery.